src/test/java/org/apache/drill_web_test_framework/web_ui/pages/OptionsPage.java

```diff
@@ -0,0 +1,244 @@
+  private WebElement parquetFilter;
+
+  @FindBy(xpath = "//button[text()='hashagg']")
+  private WebElement hashaggFilter;
+
+  @FindBy(xpath = "//button[text()='hashjoin']")
+  private WebElement hashjoinFilter;
+
+  @FindBy(xpath = "//th[text()='OPTION']")
+  private WebElement optionSortBy;
+
+  @FindBy(xpath = "//th[text()='VALUE']")
+  private WebElement valueSortBy;
+
+  @FindBy(xpath = "//th[text()='DESCRIPTION']")
+  private WebElement descriptionSortBy;
+
+  @FindBy(css = "button[onclick=\"alterSysOptionUsingId('drill.exec.functions.cast_empty_string_to_null')\"]")
+  private WebElement updateBooleanButton;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select")
+  private WebElement booleanDropdown;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select>option[value=\"true\"]")
+  private WebElement booleanDropdownTrue;
+
+  @FindBy(css = "button[onclick=\"alterSysOption('drill.exec.functions.cast_empty_string_to_null','false', 'BOOLEAN')\"]")
+  private WebElement booleanDefaultButton;
+
+  @FindBy(css = "form[id='drill.exec.memory.operator.output_batch_size']>div>div>input")
+  private WebElement numericField;
+
+  @FindBy(css = "button[onclick=\"alterSysOptionUsingId('drill.exec.memory.operator.output_batch_size')\"]")
+  private WebElement numericUpdateButton;
+
+  @FindBy(css = "button[onclick=\"alterSysOption('drill.exec.memory.operator.output_batch_size','16777216', 'LONG')\"]")
+  private WebElement numericDefaultButton;
+
+  @FindBy(css = "form[id='drill.exec.storage.file.partition.column.label']>div>div>input")
+  private WebElement stringField;
+
+  @FindBy(css = "button[onclick=\"alterSysOptionUsingId('drill.exec.storage.file.partition.column.label')\"]")
+  private WebElement stringUpdateButton;
+
+  @FindBy(css = "button[onclick=\"alterSysOption('drill.exec.storage.file.partition.column.label','dir', 'STRING')\"]")
+  private WebElement stringDefaultButton;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select>option[value='true']:checked")
+  private WebElement valueOfBooleanTrue;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select>option[value='false']:checked")
+  private WebElement valueOfBooleanFalse;
+
+  @FindBy(css = "input[value='16777217']")
+  private WebElement updatedValueOfNumeric;
+
+  @FindBy(css = "input[value='16777216']")
+  private WebElement defaultValueOfNumeric;
+
+  @FindBy(css = "input[value='test']")
+  private WebElement updatedValueOfString;
+
+  @FindBy(css = "input[value='dir']")
+  private WebElement defaultValueOfString;
+
+  @FindBy(xpath = "//div[starts-with(text(),\"Found\")]")
+  private WebElement foundMatchesOptions;
+
+  public List<List<String>> getOptionsTable() {
+    return getTable(optionTable);
+  }
+
+  public void searchFieldSendKeys(String string) {
+    searchField.clear();
```

Comment:
Use BasePage#sendText method instead.

Reply:
Method was reworked, it returns OptionsPage now

Reply:
You don't need a clear method here. sendText() will do it.

Reply:
Done